### ISSUE TITLE
duplicate project

### ISSUE THREAD

**Issue by Blackcipher101** [kossiitkgp/kwoc-bugs#5]
## Description of the Issue
My project was added twice whereas I added only once

* URL:https://kwoc.kossiitkgp.org/dashboard/mentor
* Date Observed:1/12/2020

**Comment by yashrsharma44** [comment]
Hey did you fill the form once and submit? Or any issues happened on the way?

**Comment by yashrsharma44** [comment]
Hey, were you able to submit your project in one go? Or did you face any issues in between?

**Comment by rakaar** [comment]
Duplicate of `CamSanx` project has been removed. Hence closing the issue
